client/my-sites/plugins/plugin-information/index.jsx

```diff
@@ -147,6 +148,19 @@ export default React.createClass( {
 		return {};
 	},
 
+	getDefaultActionLinks( plugin ) {
+		let adminUrl = get( this.props, 'selectedSite.options.admin_url' );
```

Comment:
Oh, inspecting `this.props` reveals that there's only a `site` prop, not a `selectedSite` one.